analyzer/src/funTest/kotlin/managers/BundlerTest.kt

```diff
@@ -67,13 +67,15 @@ class BundlerTest : WordSpec() {
                 val actualResult = createBundler().resolveDependencies(listOf(definitionFile))[definitionFile]
 
                 actualResult shouldNotBe null
```

Comment:
optional: 'actualResult shouldNotBe null' can also be remove because it is checked in the following line anyway.

Reply:
If by "following line" you mean the line containing `with(actualResult!!)`, that's actually not a check but a hard cast. And we can only assume the hard cast to work because we do the `shouldNotBe` check before.

Ideally, *kotlintest* would provide matchers that implement [contracts](https://kotlinlang.org/docs/reference/whatsnew13.html#contracts) so that after a check like `actualResult shouldNotBe null` no more hard cast is necessary.